test/e2e/common/secrets.go

```diff
@@ -180,14 +201,19 @@ func secretForTest(namespace, name string) *api.Secret {
 	}
 }
 
-func doSecretE2EWithoutMapping(f *framework.Framework, defaultMode *int32) {
+func doSecretE2EWithoutMapping(f *framework.Framework, defaultMode *int32, secret2 *api.Secret) {
```

Comment:
But the whole point is that *this* is not about two secrets. It's just one, that mounts and test. It happens to be called in several places, one of them is another test that creates a secret with the same name.

That other place is a test about two secrets. Not this.